test/k8sT/DatapathConfiguration.go

```diff
@@ -85,6 +89,82 @@ var _ = Describe("K8sDatapathConfig", func() {
 		ExpectWithOffset(1, err).To(BeNil(), "Failure while waiting for all cilium endpoints to reach ready state")
 	}
 
+	Context("MonitorAggregation", func() {
+		It("Checks that monitor aggregation restricts notifications", func() {
+			deployCilium([]string{
+				"--set global.bpf.monitorAggregation=medium",
+				"--set global.bpf.monitorInterval=60s",
+				"--set global.bpf.monitorFlags=syn",
+				"--set global.debug.enabled=false",
```

Comment:
Why does `debug.enabled` have to be false here? Is this due to complexity limits?

Reply:
No, this is because monitor aggregation is orthogonal to debug monitor logs so if debug is enabled we end up sending various other notifications and the aggregation doesn't properly work. Disabling here allows us to actually test how monitor aggregation would work in real deployments.